src/fparser/two/tests/fortran2003/test_program_stmt_r1102.py

```diff
@@ -75,3 +75,13 @@ def test_prog_symbol_table():
                         "end program my_prog\n")
     prog = Program(reader)
     assert "my_prog" in SYMBOL_TABLES._symbol_tables
+
+
+def test_get_name():
```

Comment:
Please mark the two new tests with `@pytest.mark.usefixtures("f2003_create")` as otherwise they don't work in isolation and may fail when run in parallel (e.g. `pytest fortran2003/test_program_stmt_r1102.py::test_get_name` fails currently).

Reply:
Is there a difference between using the decorator and passing the fixture as an argument? Is it just that the return value of `f2003_create` isn't needed in the tests?

Reply:
Yes, the return value isn't needed and pylint will complain about an unused variable.